gui/__tests__/containers/JobSpec.test.js

```diff
@@ -40,5 +41,30 @@ describe('containers/JobSpec', () => {
     expect(wrapper.text()).toContain('Created2018-05-10T00:41:54.531043837Z')
     expect(wrapper.text()).toContain('Run Count1')
     expect(wrapper.text()).toContain('{"value":"8400.00"}')
+    expect(wrapper.text()).not.toContain('Show More')
+  })
+
+  it('displays a show more link if there are more than 5 runs', async () => {
+    const runs = [
```

Comment:
Could we use the injection trick here to reduce the verbosity of this test?

Reply:
@j16r this one is a higher level integration test.